packages/perseus/src/util/extract-perseus-data.test.ts

```diff
@@ -89,3 +94,55 @@ describe("injectWidgets", () => {
 
     // Add more test cases for other widget types...
 });
+
+describe("getImagesWithoutAltData", () => {
+    describe("image", () => {
```

Comment:
Is the idea that in the future we might have more `describe` blocks for different widgets? If so, I might rename this to `image widget`. Otherwise I wonder if we could just drop this layer.